src/collective/cover/browser/cover.py

```diff
@@ -271,7 +323,7 @@ def render(self):
         uid = self.request.form.get('uid')
         if tile_type and tile_id and uid:
             tile = self.context.restrictedTraverse('{0}/{1}'.format(tile_type, tile_id))
-            if isinstance(tile, ListTile):
+            if IListTile.providedBy(tile):
```

Comment:
I double check this and it works fine like this (it provides IListTile for tile List and tile Carousel)